### PR TITLE
Loadbalancer integration into new controlplane

### LINKED ISSUE #89: Support for load-balancing policies in the new controlplane

### PR DESCRIPTION
**Pull request by zivnevo**
Fixes #89 

For now, using `api.Policy` to also pass load-balancing policies all the way from client to policy-engine (similar to access policies).
We may consider changing this later, by introducing a new API type.

### CODE REVIEW

File: pkg/policyengine/PolicyDispatcher.go [hunk 1]
```diff
@@ -214,11 +214,29 @@ func connPolicyFromBlob(blob io.Reader) (*policytypes.ConnectivityPolicy, error)
...
+	lbPolicy := &LBPolicy{}
+	err := json.NewDecoder(blob).Decode(lbPolicy)
+	if err != nil {
+		plog.Errorf("failed decoding load-balancing policy: %v", err)
+		return nil, err
+	}
+	return lbPolicy, nil
+}
+
+func (pH *PolicyHandler) AddLBPolicy(policy *api.Policy) error {
```
Comment (kfirtoledo) [inline pkg/policyengine/PolicyDispatcher.go:227]: Can you put a comment before those functions.

File: pkg/controlplane/instance.go [hunk 2]
```diff
@@ -593,6 +657,7 @@ func NewInstance(peerTLS *util.ParsedCertData, storeManager store.Manager, platf
 		imports:       imports,
 		bindings:      bindings,
 		policies:      policies,
```
Comment (kfirtoledo) [inline pkg/controlplane/instance.go:659]: Maybe to change it to acPolicies

**Review by kfirtoledo — COMMENTED** [review]
LGTM, some small comments

File: pkg/policyengine/PolicyDispatcher.go (hunk 1 above)
Comment (zivnevo) [reply pkg/policyengine/PolicyDispatcher.go:230]: Done

File: pkg/controlplane/instance.go (hunk 2 above)
Comment (zivnevo) [reply pkg/controlplane/instance.go]: Done

**Review by kfirtoledo — APPROVED** [review]

**Pull request merged**